components/application/widgets/TodoList.js

```diff
@@ -7,7 +7,7 @@ import PopupMenu, { useMenu } from "./PopupMenu";
 
 function TodoList({
   tasks,
-  title,
+  title, showProject,
```

Comment:
The parameters here are placed in a single line, which differs from the arrangement of other parameters. Is there any specific reason for this? I would recommend aligning them in the same manner as the other parameters for consistency.

Reply:
fixed